Gruntfile.coffee

```diff
@@ -110,6 +113,7 @@ module.exports = (grunt) ->
 
   # register tasks
   grunt.registerTask "build", ["clean", "coffeelint", "coffee", "concat"]
+  grunt.registerTask "watchDev", ["clean", "coffee"]
```

Comment:
I use this when I want to just build code - (do not run any tests) so I can run export script by myself. `grunt test` is doing also linting and it is running all tests which takes too long